apps/librelingo_json_export/librelingo_json_export/export.py

```diff
@@ -20,9 +20,16 @@ def _ensure_output_dir(output_file_path):
     output_file_path.parent.mkdir(parents=True, exist_ok=True)
 
 
-def _save_as_json_file(data, output_path):
-    _ensure_output_dir(output_path)
-    with open(output_path, "w", encoding="utf-8") as f:
+def _prepare_output_path(output_file_path, settings):
+    res = Path(os.devnull)
+    if not _is_dry_run(settings):
+        res = output_file_path
+        _ensure_output_dir(res)
+    return res
```

Comment:
```suggestion
def _prepare_output_path(output_file_path, settings):
    if settings.dry_run:
        return Path(os.devnull)

    _ensure_output_dir(output_file_path)
    return output_file_path
```

How about that? You can save one function and have an easier to read structure?